config/clients/java/template/OpenFgaApiTest.java.mustache

```diff
@@ -0,0 +1,36 @@
+{{>licenseInfo}}
+
+package dev.openfga.api;
+
+import dev.openfga.api.invoker.*;
+import dev.openfga.api.model.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.junit.Assert.assertEquals;
+
+public class OpenFgaApiTest {
```

Comment:
TODO: Check conventions on other Auth0 projects using Gradle. This is an integration test. It shouldn't be necessary to run an OpenFGA server to do  local dev.

Probably this should have a different class name, and also a different Gradle task that isn't part of `check`. (So: changes here, in github workflow, and Makefile)

Reply:
done